backend/db/models/database.js

```diff
@@ -0,0 +1,68 @@
+const sqlite = require('sqlite3')
+const path = require('path')
+const readline = require('readline')
+const dbPath = path.resolve(__dirname, 'tables.db')
+
+//open database
+function openDB(){
+    let db = new sqlite.Database(dbPath, sqlite.OPEN_READWRITE, (err) =>{
+        if(err){
+            console.error(err.message)
+        }
+        else{
+            console.log('Connected to the tables database.')
+        }
+    })
+    return db
+}
+
+//create Board in database
+function newBoard(title){
+    let db = openDB()
+    db.serialize(function(){
+        db.run("INSERT INTO Boards (title) VALUES ('"+title+"')", (err)=>{
+            if(err){
+                console.error(err.message)
+            }
+            else{
+                console.log('Inserted into table Boards')
+            }
+        })
+    })
+
+    db.close()
+}
+
+//edit Board in database
+function editBoard(oldName, newName){ //change needed: update query so that only authorized users can make changes
```

Comment:
not really needed. sa express na na nga side nga i ensure nga authorized na ang user ig ka query ana. never jud na siya maabot dha nga need ka ug authentication igka edit sa board.